content/_includes/FusionPoolsCommonContent.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+Add SSDs to the new **Metadata VDev** and select the same layout as the **Data VDevs**.
+
+{{< hint warning >}}
+The metadata special VDEV is critical for pool operation and data integrity, so you must protect it with hot spare(s).
+{{< /hint >}}
+
+{{< expand "UPS Recommendation" "v" >}}
+When using SSDs with an internal cache, add uninterruptible power supply (UPS) to the system to help minimize the risk from power loss.
+{{< /expand >}}
+
+Using special VDEVs identical to the data VDEVs (so they can use the same hot spares) is recommended, but for performance reasons you can make a different type of VDEV (like a mirror of SSDs).
+In that case you must provide hot spare(s) for that drive type as well. Otherwise, if the special VDEV fails and there is no redundancy, the pool becomes corrupted and prevents access to stored data.
```

Comment:
```suggestion
In that case, you must provide hot spare(s) for that drive type as well. Otherwise, if the special VDEV fails and there is no redundancy, the pool becomes corrupted and prevents access to stored data.
```